src/applications/appeals/996/components/ConfirmationPageV2.jsx

```diff
@@ -68,24 +70,27 @@ export const ConfirmationPageV2 = () => {
     false;
   const { informalConferenceRep = {} } = data;
 
+  const submitDate = getReadableDate(
+    submission?.timestamp || new Date().toISOString(),
+  );
+
   const conferenceMessage = hasConference
     ? `Since you requested an informal conference, we’ll contact ${
         hasRepContact ? 'your accredited representative' : 'you'
-      } to schedule your conference.`
+      } to schedule your conference. `
```

Comment:
Extra space?

Reply:
I'd rather remove it now and wait for CI than create a PR just to remove a space 😛